back/assets/myoback_assets.xml

```diff
@@ -0,0 +1,1324 @@
+    </visual>
+
+  <statistic meansize="0.15762" extent="2.51911" center="0 0 0.259556"/>
+  <default class="main">
+    <joint limited="true" armature="1e-05" damping="0.05"/>
+    <site size="0.001 0.005 0.005"/>
+    <tendon limited="false" width="0.005" rgba="0.95 0.3 0.3 1"/>
+    <default class="muscle">
+      <general ctrllimited="true" ctrlrange="0 1" dyntype="muscle" gaintype="muscle" biastype="muscle" dynprm="0.01 0.04 0 0 0 0 0 0 0 0" gainprm="0.75 1.05 -1 200 0.5 1.6 1.5 1.3 1.2 0" biasprm="0.75 1.05 -1 200 0.5 1.6 1.5 1.3 1.2 0"/>
+    </default>
+    <default class="motor">
+      <general gainprm="5 0 0 0 0 0 0 0 0 0"/>
+    </default>
+  </default>
+
+    <asset>
+    <mesh name="pelvis_geom_1_pelvis_rv" file="../myo_sim/meshes/r_pelvis.stl"/>
+    <mesh name="pelvis_geom_2_pelvis_lv" file="../myo_sim/meshes/l_pelvis.stl"/>
+    <mesh name="sacrum_geom_1_sacrum" file="../myo_sim/meshes/sacrum.stl"/>
+    <mesh name="lumbar5_geom_1_lumbar5" file="../myo_sim/meshes/lumbar5.stl" scale="0.87 0.87 0.87"/>
+    <mesh name="lumbar4_geom_1_lumbar4" file="../myo_sim/meshes/lumbar4.stl" scale="0.87 0.87 0.87"/>
+    <mesh name="lumbar3_geom_1_lumbar3" file="../myo_sim/meshes/lumbar3.stl" scale="0.87 0.87 0.87"/>
+    <mesh name="lumbar2_geom_1_lumbar2" file="../myo_sim/meshes/lumbar2.stl" scale="0.87 0.87 0.87"/>
+    <mesh name="lumbar1_geom_1_lumbar1" file="../myo_sim/meshes/lumbar1.stl" scale="0.87 0.87 0.87"/>
+    <mesh name="torso_geom_1_thoracic12_s" file="../myo_sim/meshes/thoracic12_s.stl" scale="0.87 0.87 0.87"/>
+    <mesh name="torso_geom_2_thoracic11_s" file="../myo_sim/meshes/thoracic11_s.stl" scale="0.87 0.87 0.87"/>
+    <mesh name="torso_geom_3_thoracic10_s" file="../myo_sim/meshes/thoracic10_s.stl" scale="0.87 0.87 0.87"/>
+    <mesh name="torso_geom_4_thoracic9_s" file="../myo_sim/meshes/thoracic9_s.stl" scale="0.87 0.87 0.87"/>
+    <mesh name="torso_geom_5_thoracic8_s" file="../myo_sim/meshes/thoracic8_s.stl" scale="0.87 0.87 0.87"/>
+    <mesh name="torso_geom_6_thoracic7_s" file="../myo_sim/meshes/thoracic7_s.stl" scale="0.87 0.87 0.87"/>
+    <mesh name="torso_geom_7_thoracic6_s" file="../myo_sim/meshes/thoracic6_s.stl" scale="0.87 0.87 0.87"/>
+    <mesh name="torso_geom_8_thoracic5_s" file="../myo_sim/meshes/thoracic5_s.stl" scale="0.87 0.87 0.87"/>
+    <mesh name="torso_geom_9_thoracic4_s" file="../myo_sim/meshes/thoracic4_s.stl" scale="0.87 0.87 0.87"/>
+    <mesh name="torso_geom_10_thoracic3_s" file="../myo_sim/meshes/thoracic3_s.stl" scale="0.87 0.87 0.87"/>
+    <mesh name="torso_geom_11_thoracic2_s" file="../myo_sim/meshes/thoracic2_s.stl" scale="0.87 0.87 0.87"/>
+    <mesh name="torso_geom_12_thoracic1_s" file="../myo_sim/meshes/thoracic1_s.stl" scale="0.87 0.87 0.87"/>
+    <mesh name="torso_geom_13_ribcage_s" file="../myo_sim/meshes/ribcage_s.stl" scale="0.87 0.87 0.87"/>
+    <mesh name="lower_exo_belt_geom_1" file="../myo_sim/meshes/LowerExoBelt.stl" scale="0.0013 0.0013 0.0013"/>
```

Comment:
should those assets be moved into a `myoback_assets_exosuit`?

Reply:
ff16d3416eba6a2de3bc440c76bac588cf5bfdbb
is this what you have in mind?

Reply:
in terms of muscle attaching outside of the body, is this what you are looking for? (using the abdomen muscle as an example)
![image](https://github.com/user-attachments/assets/2168b4a7-6891-41ec-9df7-333c5009082c)
Image before adjustments for comparison:
![image](https://github.com/user-attachments/assets/3683c43c-d837-48be-a793-e26829be3f49)